### PR TITLE
clarifying 03-intermediate/03-linkdrop.md

### PR DESCRIPTION
**Pull request by ryancwalsh**
See the original PR at https://github.com/near/sdk-docs/pull/120

This PR rewords some sections in ways that I think would have made this lesson easier for me to understand.

But please double-check the changes for accuracy.

Thanks!

### CODE REVIEW

File: docs/3.tutorials/crosswords/03-intermediate/03-linkdrop.md [hunk 1]
```diff
@@ -26,10 +26,12 @@ Let's discuss how this testnet account gets created.
 Notice the new account will end in `.testnet`. This is because the account `testnet` will create a subaccount (like we learned about [earlier in this tutorial](../01-basics/02-add-functions-call.md#create-a-subaccount)) called `vacant-name.testnet`.
 
 There are two ways to create this subaccount:
+
 1. Use a full-access key for the account `testnet` to sign a transaction with the `CreateAccount` Action.
-2. Have a smart contract deployed to the `testnet` account that has a Promise executing the `CreateAccount` Action. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+2. In a smart contract deployed to the `testnet` account, call a Promise executing the `CreateAccount` Action. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+
+(In the example below that uses NEAR CLI to create a new account, it's calling the Promise on the linkdrop contract that is deployed to the top level "near" account on mainnet.)
```
Comment (idea404) [inline docs/3.tutorials/crosswords/03-intermediate/03-linkdrop.md:33]: I wouldn't say the promise is being called. The function is called, which is async, which creates a promise. But in general one calls functions that execute logic. If this is performed asynchronously then promises are created

**Comment by ryancwalsh** [comment]
@idea404 Thanks. How does it look now?

**Review by idea404 — APPROVED** [review]
LGTM

File: docs/3.tutorials/crosswords/03-intermediate/03-linkdrop.md [hunk 2]
```diff
@@ -26,10 +26,12 @@ Let's discuss how this testnet account gets created.
 Notice the new account will end in `.testnet`. This is because the account `testnet` will create a subaccount (like we learned about [earlier in this tutorial](../01-basics/02-add-functions-call.md#create-a-subaccount)) called `vacant-name.testnet`.
 
 There are two ways to create this subaccount:
+
 1. Use a full-access key for the account `testnet` to sign a transaction with the `CreateAccount` Action.
-2. Have a smart contract deployed to the `testnet` account that has a Promise executing the `CreateAccount` Action. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+2. In a smart contract deployed to the `testnet` account, call the `CreateAccount` Action, which is an async method that returns a Promise. (More info about writing a [`CreateAccount` Promise](/sdk/rust/promises/create-account).)
+
+(In the example below that uses NEAR CLI to create a new account, it's calling CreateAccount on the linkdrop contract that is deployed to the top level "near" account on mainnet.)
```
Comment (bucanero) [inline docs/3.tutorials/crosswords/03-intermediate/03-linkdrop.md:33]:
```suggestion
(In the example below that uses NEAR CLI to create a new account, it's calling `CreateAccount` on the linkdrop contract that is deployed to the top level "near" account on mainnet.)
```

**Pull request merged**